src/components/Lesson/Progress.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import withStyles from 'isomorphic-style-loader/lib/withStyles';
+import styles from './Progress.scss';
+import ProgressBar from 'react-bootstrap/lib/ProgressBar';
+
+const Progress = ({checkedCheckboxes, totalCheckboxes}) => {
+  const now = totalCheckboxes > 0 ? 100 * checkedCheckboxes / totalCheckboxes : 0;
+  const bsStyle = now < 100 ? 'info' : 'success';
+  const className = styles.marginTop;
```

Comment:
Kanskje kalle `marginTop` for `progressBar` eller noe? For du har jo en mediaquery inni der...